e2e/cypress/integration/02-apd/01-apd-basics.cy.js

```diff
@@ -1342,6 +1342,68 @@ describe('APD Basics', { tags: ['@apd', '@default'] }, () => {
     });
   });
 
+  describe('Accessibility Tests', () => {
```

Comment:
maybe instead of clicking continue you could click on the item in the side panel, just to make it less fragile if the order changes

Reply:
changed all `cy.contains('Continue').click()` to `cy.goToX.click()`